### ISSUE TITLE
Issue running le-hex-to-ip.py script on mac

### ISSUE THREAD

**Issue by Mastermjr** [clausing/scripts#3]
apparently python must not support the format string %prog on mac.... its only grabbing the `%p` character.  So i deleted this format string. 

```
Traceback (most recent call last):
  File "./le-hex-to-ip.py", line 38, in <module>
    main()
  File "./le-hex-to-ip.py", line 18, in main
    args = parser.parse_args()
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 1825, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 1858, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 2095, in _parse_known_args
    self.error(_('the following arguments are required: %s') %
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 2580, in error
    self.print_usage(_sys.stderr)
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 2550, in print_usage
    self._print_message(self.format_usage(), file)
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 2516, in format_usage
    return formatter.format_help()
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 283, in format_help
    help = self._root_section.format_help()
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 214, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 214, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/local/Cellar/python@3.9/3.9.17/Frameworks/Python.framework/Versions/3.9/lib/python3.9/argparse.py", line 300, in _format_usage
    usage = usage % dict(prog=self._prog)
ValueError: unsupported format character 'p' (0x70) at index 8
```

also, /bin/env doesn't exist on macos :( refactored that to `/usr/bin/env python3`

I don't know what testing you do (if any), and it was pretty easy to fix on my system, but thought you might find the above interesting ^^. So feel free to close this or leave it open :) 

Thanks for the script and the blog, good to know I can use the /proc/<pid>/ to hunt for hidden tcp connections, thats pretty neat!

**Comment by clausing** [comment]
I changed both of them, on my current Ubuntu systems, /bin is a soft-link to /usr/bin anyway.

**Comment by clausing** [comment]
Closed with v1.1.1
